scripts/skills/perks/perk_rf_poise.nut

```diff
@@ -102,4 +109,14 @@ this.perk_rf_poise <- ::inherit("scripts/skills/skill", {
 			_properties.OffensiveReachIgnore += 1;
 		}
 	}
+
+	q.getStaminaModifierThreshold <- function()
+	{
+		local b = this.getContainer().getActor().getBaseProperties().getClone();
+		foreach (trait in this.getContainer().getSkillsByFunction(@(s) s.isType(::Const.SkillType.Trait) || s.isType(::Const.SkillType.PermanentInjury)))
+		{
+			trait.onUpdate(b);
+		}
+		return ::Math.max(0, 0.35 * b.Stamina * (b.StaminaMult >= 0 ? b.StaminaMult : 1.0 / b.StaminaMult));
```

Comment:
the `0.35` should be a member variable for easier changing by future balancing commits or submods.